src/main/tut/index.html

```diff
@@ -205,6 +237,7 @@
 
 ## Enhancement without inheritance!
 
+<!-- COMMENT -->
 Important realisation: _we've added a new feature to `Int`, even though it's `final`_.
```

Comment:
Good point, feature is absolutely not the right term here. I'll replace it with _behaviour_ and potentially expand on it orally if necessary. Thanks!